network/message/message.proto

```diff
@@ -4,13 +4,13 @@ package message;
 
 // Run protoc --gofast_out=. message.proto to generate message.pb.go
 
-// Message models a single message that is supposed to get exchanged by the gossip network
+// Message models a single message that is supposed to get exchanged by the
+// gossip network
 message Message {
-    string ChannelID = 1;
-    bytes EventID = 2;
-    bytes OriginID = 3;
-    repeated bytes TargetIDs = 4;
-    bytes Payload = 5;
-    string Type = 6;
+  string ChannelID = 1;
+  bytes EventID = 2;
+  bytes OriginID = 3 [deprecated = true];
```

Comment:
If we _just_ deprecate this, we're not removing the field and are [still allowing its use](https://stackoverflow.com/a/60280361).

Which means, technically, that we could still have `OriginID` claims in this field, which would still be ser/deser-ialized, and that we should still report if different from the result of the validation logic.

Is it possible to just remove this field and mark the release that would include this PR as breaking? I think it's worth it.

/cc @ramtinms @vishalchangrani for advice on the Message format change.

Reply:
Yes, although I'm hoping we can eventually [remove all of these fields](https://github.com/onflow/flow-go/blob/smnzhu/runnable-flip/flips/network-api.md) except `Payload`, and I think this is something that can wait until later and we can remove them all in one fell swoop.

Reply:
OK. Do we have an issue for that?

Reply:
Not exactly, but it is part of https://github.com/dapperlabs/flow-go/issues/5847

Reply:
As long as the network layer is passing the application layer the origin ID (which we are), it doesn't matter if we remove the field now or later.